tests/cypress/support/commands.ts

```diff
@@ -231,7 +230,7 @@ Cypress.Commands.add('nameSpaceMenuToggle', (namespaceName) => {
     cy.get('.top > .ns-filter').click();
   }
   cy.get('div.ns-item').contains(namespaceName).scrollIntoView()
-  cy.get('div.ns-item').contains(namespaceName).click()
+  cy.get('div.ns-item').contains(namespaceName).click({ force: true })
```

Comment:
Are we sure this won't affect current tests in other lanes?

Reply:
I don't see any weird behavior while upgrading from 2.9 to 2.10 and 2.10 to 2.11 (head).
Okay, I will remove it just for cleaner upgrade process.

Reply:
Done